admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -10,20 +10,37 @@ is usually not necessary to edit ``config/config.php``.
 
 .. note:: The installer creates a configuration containing the essential parameters.
    Only manually add configuration parameters to ``config/config.php`` if you need to
-   use a special value for a parameter. **Do not copy everything from**
-   ``config/config.sample.php`` **. Only enter the parameters you wish to modify!**
+   use a special value for a parameter.
+   **Do not copy everything from** ``config/config.sample.php``
+   **. Only enter the parameters you wish to modify!**
+
+Multiple configuration files
+----------------------------
 
 ownCloud supports loading configuration parameters from multiple files.
-You can add arbitrary files ending with :file:`.config.php` in the :file:`config/`
-directory, for example you could place your email server configuration
-in :file:`email.config.php`. This allows you to easily create and manage
-custom configurations, or to divide a large complex configuration file 
-into a set of smaller files. These custom files are not overwritten by 
-ownCloud, and the values in these files take precedence over :file:`config.php`.
+You can add arbitrary files ending with `.config.php` in the `config/` directory.
+
+Example:
+
+You could place your email server configuration in `email.config.php`. 
+
+This allows you to easily create and manage custom configurations, or to divide a 
+large complex configuration file into a set of smaller files. 
+
+.. note:: These custom files are not overwritten by ownCloud, 
+   and the values in these files take precedence over `config.php`
+
+.. note:: ownCloud may write configurations into the main ``config.php`` file.
+   This parametrisation may confict with same keys but different settings from additional config files.
+   Be carefule when using this capability!
 
 .. The following section is auto-generated from 
 .. https://github.com/owncloud/core/blob/master/config/config.sample.php
-.. Do not edit this file; edit the source file in core
+.. Do not edit the content of this file between _section_start and _sections_end
+.. The content there will be loaded and replaced via script from the source file from link above
```

Comment:
s/via/by a/g

Reply:
I'd suggest the following instead:

The content between these markers will be replaced by a script.